### PR TITLE
run WML tools, take 2

### PR DESCRIPTION
**Pull request by cooljeanius**
This is basically just another attempt at #4, except this time I didn't run `wmlindent`. I'm not marking it as a draft this time, because the campaign now loads properly for me.

### CODE REVIEW

File: utils/misc.cfg [hunk 1]
```diff
@@ -50,7 +50,7 @@
 			{CONDITION_STRUCK}
 		[/show_if]
 		{BONUS_OBJECTIVE_CAPTION}
-		description="<span strikethrough='true'>" + {TEXT} + "</span>"
+		description=_ "<span strikethrough='true'>" + {TEXT} + "</span>"
```
Comment (CelticMinstrel) [inline utils/misc.cfg:53]: This is incorrect. The only translatable part is `{TEXT}`.

Comment (cooljeanius) [reply utils/misc.cfg]: ok, fixed in [28ab68d](https://github.com/CelticMinstrel/Black_Cross_Of_Aleron/pull/5/commits/28ab68dcfeebe93edef2953e4b25cffab9c2a125)

Comment (cooljeanius) [reply utils/misc.cfg]: There, now that I've done [21e7813](https://github.com/CelticMinstrel/Black_Cross_Of_Aleron/pull/5/commits/21e781327e42bc7beec1ffd266864f3410d829a7), *now* I can resolve this conversation...

**Comment by cooljeanius** [comment]
ok I updated past [fcf257c](https://github.com/CelticMinstrel/Black_Cross_Of_Aleron/commit/fcf257cf4052c2d1f855463dc08c9f3ad96c886a); regenerating the potfile with my own copy of `wmlxgettext` after that still results in a massive diff anyways, though, due to case sensitivity ("of" vs. "Of"), so I'm just leaving that out for now. (This PR will probably need to be squashed before merging, btw)

**Pull request merged**